src/participants/trigger_service.py

```diff
@@ -174,12 +177,28 @@ def handle_wi(self, wid):
             #self.obs.getCreatePackage(str(project), str(package))
         #else:
         try:
-            core.show_files_meta(self.obs.apiurl, str(project), str(package), expand=False, meta=True)
+            pkginfo=core.show_files_meta(self.obs.apiurl, str(project), str(package), expand=False, meta=True)
```

Comment:
Given that 78ad45f followed the broken  8d0bd4c, does that maybe explain the weird behaviour contradicting the obs documentation?